lib/reviews.js

```diff
@@ -53,6 +53,9 @@ function extractReviews (xml) {
   };
 
   const list = xml.feed.entry;
+  if (!list) {
+    return []
```

Comment:
it's easier to just do `const list = xml.feed.entry || [];`

plus this is missing a semicolon and that makes the build fail